rollout/trafficrouting/alb/alb_test.go

```diff
@@ -463,6 +463,14 @@ func (f *fakeAWSClient) getAlbStatus() *v1alpha1.ALBStatus {
 func TestVerifyWeight(t *testing.T) {
 	newFakeReconciler := func(status *v1alpha1.RolloutStatus) (*Reconciler, *fakeAWSClient) {
 		ro := fakeRollout(STABLE_SVC, CANARY_SVC, nil, "ingress", 443)
+		ro.Status.StableRS = "a45fe23"
+		ro.Spec.Strategy.Canary.Steps = []v1alpha1.CanaryStep{{
+			SetWeight:      pointer.Int32Ptr(10),
+			Pause:          nil,
+			Experiment:     nil,
+			Analysis:       nil,
+			SetCanaryScale: nil,
```

Comment:
Same thing here as well. In general it's a good practice to minimize the LOCs of code you are putting so don't put anything unless it's needed :)